src/scss/components/_register-comp.scss

```diff
@@ -0,0 +1,204 @@
+.form {
+  background: rgb(255, 206, 70);
```

Comment:
Use the hex value of the color. Check if this color already exists in the variables, otherwise add it in order to be used across the site.

Reply:
Done!